.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "twilio-react"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2019,
+    "sourceType": "module", 
+    "ecmaFeatures": { 
+    "jsx": true
+   }
+  },
+  "rules": {
+    "no-console": "off",
+    "no-alert": "off",
+    "import/no-extraneous-dependencies": "off",
+    "no-use-before-define": "off",
```

Comment:
I actually believe this should be enabled (and disable them per line if needed)? @mythilytm